app/views/account/users/index.html.slim

```diff
@@ -0,0 +1,9 @@
+h1 All Users
+.container
+  .table-responsive
+    table.table.table-bordered
+      th Email
+      th Name
+      th Role
+      = render @users
+  = paginate @users
```

Comment:
add bootstrap styles (kaminari has gem for it ... something like kaminary-bootstrap)